packages/extension-table/package.json

```diff
@@ -24,7 +24,7 @@
     "@tiptap/core": "^2.0.0-beta.1"
   },
   "dependencies": {
-    "@_ueberdosis/prosemirror-tables": "1.1.3",
+    "@_ueberdosis/prosemirror-tables": "1.2.2",
```

Comment:
We were using a temporary package here to fix typescript dependencies a bit back.
`@_ueberdosis/prosemirror-tables` won't work as we don't have that version published on npm.

I guess you wanted to change to `prosemirror-tables@1.2.2`? if you do this, can you test it and let me know? I'll take another look at your PR then.

Reply:
That is indeed exactly what I meant. I have made the changes, tested it locally, and pushed it to GitHub.

![image](https://user-images.githubusercontent.com/13137042/185923278-a9f3777e-d9ea-4b7b-8985-8dbc98c1dbb4.png)
![image](https://user-images.githubusercontent.com/13137042/185923316-2158a508-2347-4723-8a47-bf0aa5418706.png)

For some reason, the GitHub tests keep failing though.

Reply:
I'll take a look at them! :) Thanks for reacting so quickly!

Reply:
Thanks to you as well!

I had no idea how to make the code pass the GitHub "building phase", so you might want to ignore my third commit (c8be86c). Cypress keeps throwing timeouts.

Hope it gets resolved soon 🤞